### PR TITLE
lithuanian translation of json

### PR DESCRIPTION
**Pull request by zygimantus**

### CODE REVIEW

File: lt-lt/json.html.markdown [hunk 1]
```diff
@@ -0,0 +1,80 @@
+---
+language: json
+filename: learnjson.json
```
Comment (vendethiel) [inline lt-lt/json.html.markdown]: please add `-lt` to filename

File: lt-lt/json.html.markdown [hunk 2]
```diff
@@ -0,0 +1,80 @@
...
+contributors:
+  - ["Zygimantus", "https://github.com/zygimantus"]
```
Comment (vendethiel) [inline lt-lt/json.html.markdown]: please add a `lang`

**Comment by zygimantus** [comment]
Is it OK now?

**Comment by vendethiel** [comment]
Almost: You left in the old file and didn't add a "lang: lt" key :)

**Comment by zygimantus** [comment]
Try now :)

File: lt-lt/json-lt.html.markdown [hunk 3]
```diff
@@ -0,0 +1,81 @@
+---
+language: json
+filename: learnjson.json
```
Comment (vendethiel) [inline lt-lt/json-lt.html.markdown]: Almost! this one needs the -lt as well. Thanks!

**Comment by zygimantus** [comment]
Like this?

**Comment by vendethiel** [comment]
Thanks! :cake:

**Pull request merged**